frontend/packages/patternfly/src/components/notification-drawer/notification-drawer.tsx

```diff
@@ -15,7 +15,7 @@ const NotificationDrawer: React.FC<NotificationDrawerProps> = ({
   notificationEntries,
   className,
 }) => {
-  const panelContent = (
+  const panelContent = isExpanded && (
```

Comment:
this is required because an empty render of `DrawerPanelContent` lingers when the panel is not expanded in the `Drawer`
![drawecontentflaw](https://user-images.githubusercontent.com/38663217/86797660-8e631000-c08d-11ea-9c3a-d760ba51c525.gif)

Reply:
Raised a bug on patternfly: https://github.com/patternfly/patternfly-react/issues/4497

Reply:
Raised another bug on patternfly: https://github.com/patternfly/patternfly-react/issues/4498

Reply:
Fine for now until patternfly fixes the issue.

Reply:
We should look at updating to new patternfly version.
But we can do so in another PR and ensure to revert this. Therefore add a comment saying to remove this fix when patternfly version is bumped.